run_test.sh

```diff
@@ -23,23 +23,28 @@ function run_npm_test {
   cd ..
 }
 
-# Tests if lint is required on client side code
-function run_npm_lint_test {
+# Tests if lint is required
+function run_lint_test {
   cd static
   npm list eslint || npm install eslint
   if ! npm run test-lint; then
     echo "\nFix lint errors by running ./run_test.sh -f"
     exit 1
   fi
   cd ..
+  if ! yapf --recursive --diff --style=google server/ tools/; then
```

Comment:
need to install yapf first?

Reply:
updated the script to install if not found